src/client/producer.rs

```diff
@@ -440,11 +440,13 @@ where
                 return;
             }
         };
-        if output.is_empty() {
-            return;
-        }
 
-        let r = client.produce(output).await;
+        let r = if output.is_empty() {
```

Comment:
I think this could occur if someone flushed an empty BatchProducer??

Reply:
done